### PR TITLE
Replace unpkg.com API with jsDelivr API in file browser

### PR DESCRIPTION
**Pull request by MartinKolarik**
Replaces usage of unpkg.com API with jsDelivr API for the file browser as previously discussed, because it is more stable and built for production usage. The API is also versioned which means there are never breaking changes such as #614.

### CODE REVIEW

**Comment by Haroenv** [comment]
Deploy preview ready!

Built with commit dbdb5495bd817452fb9c38ae97f3518fabdde688

https://deploy-preview-727--yarnpkg.netlify.com

File: js/src/lib/Details/FileBrowser.js [hunk 1]
```diff
@@ -28,8 +24,16 @@ export default class FileBrowser extends React.PureComponent {
 
   _fetchFiles() {
     this.setState({ error: null, files: null });
-    const url = this._getBaseURL() + '?meta';
+    const url = `https://data.jsdelivr.com/v1/package/npm/${this.props.objectID}@${this.props.version}`;
     fetch(url)
+      .then(response => {
+        if (!response.ok) {
```
Comment (Daniel15) [inline js/src/lib/Details/FileBrowser.js]: Shouldn't error handling be in ` catch` block, or as the second argument to `then`? Does jsdelivr not use proper HTTP codes for their responses?

File: js/src/lib/Details/FileBrowser.js [hunk 2]
```diff
@@ -28,8 +24,16 @@ export default class FileBrowser extends React.PureComponent {
 
   _fetchFiles() {
     this.setState({ error: null, files: null });
-    const url = this._getBaseURL() + '?meta';
+    const url = `https://data.jsdelivr.com/v1/package/npm/${this.props.objectID}@${this.props.version}`;
```
Comment (Daniel15) [inline js/src/lib/Details/FileBrowser.js]: Would prefer if this was pulled out into a separate function (eg. `getURLForPackageMetadata` or something like that)

File: js/src/lib/Details/FileBrowser.js [hunk 3]
```diff
@@ -28,8 +24,16 @@ export default class FileBrowser extends React.PureComponent {
...
     fetch(url)
+      .then(response => {
+        if (!response.ok) {
+          return response.json().then(body => {
+            throw body;
```
Comment (Haroenv) [inline js/src/lib/Details/FileBrowser.js]: I don’t understand why it’s thrown

File: js/src/lib/Details/FileBrowser.js (hunk 1 above)
Comment (MartinKolarik) [reply js/src/lib/Details/FileBrowser.js]: This was a surprise to me as well. `fetch()` [doesn't throw based on status code](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Checking_that_the_fetch_was_successful) but only if there was a network error. jsDelivr errors have the appropriate status code, and a message in the body. For a unified error handling, we need to parse the body and throw it as an error ourselves.

**Comment by MartinKolarik** [comment]
@Haroenv there's modification time and sha256 hash but not sure if those two are useful here.

File: js/src/lib/Details/FileBrowser.js (hunk 3 above)
Comment (MartinKolarik) [reply js/src/lib/Details/FileBrowser.js]: See my comment above.

**Pull request merged**